src/candidates/profile/shared/NavLinks.vue

```diff
@@ -42,6 +42,11 @@ export default {
           ico: 'code',
           exact: true,
           text: 'Knowledge tests'
+        },
+        { route: { name: 'CvUpload' },
```

Comment:
I would suggest to hide this behind the environment var, so we can enable it only for development environment for now...

For example:

```js
export default {
  data () {
    const cvUploadRoute = { ... }; // route here
    const cvUpload = (process.env.ENABLE_CV_UPLOAD === true) ? [cvUploadRoute] : [];
    return {
      menu: [
        ..., // other routes
        ...cvUpload
      ]
    };
  },
  ...
```

(or in some better way :slightly_smiling_face:)

Reply:
f841d18b1abb1ade8608cf1d1bd7241b200543ce